src/styles/main.scss

```diff
@@ -1,3 +1,78 @@
+$cell-size: 100px;
+$gap-space: 1px;
+$calendar-size: $cell-size * 7 + $gap-space * 6;
+$cell-color: #eee;
+$cell-hover-color: pink;
```

Comment:
It's proper to write colors using HEX format
You can create separate folders for variables and normalize